src/Services/HeadlessH5PService.php

```diff
@@ -495,18 +495,10 @@ public function deleteLibrary($id): bool
     {
         $library = H5pLibrary::findOrFail($id);
 
-        // TODO: check if runnable
-        // TODO: check is usable, against content. If yes should ne be deleted
-
-        // Error if in use
-        // TODO implement getLibraryUsage, once content is ready
-        // $usage = $this->getRepository()->getLibraryUsage($library);
-        /*
-        if ($usage['content'] !== 0 || $usage['libraries'] !== 0) {
-            return redirect()->route('h5p.library.index')
-                ->with('error', trans('laravel-h5p.library.used_library_can_not_destoroied'));
+        $libraryUsage = $this->getRepository()->getLibraryUsage($library->getKey());
+        if ($library->runnable || $libraryUsage['content'] > 0 || $libraryUsage['libraries'] > 0) {
```

Comment:
* `runnable` that shound't be checked, as fas as i know this value means that you create create content using this library 
* `$libraryUsage['libraries'] > 0` this shouldn't be checked as well, if there is no content for a given lib, delete request should delete library and it's dependcies